### DISCUSSION TITLE
Lua Filter for Obsidian Markdown?

### DISCUSSION THREAD

**Discussion by UnicornBirb** [kdheepak/kdheepak.github.io#14] [Q&A]
Hey, ran across your repo where you posted your lua filter `https://github.com/kdheepak/blog/blob/main/pandoc/alert.lua`. If I wanted to use that filter for [Obsidian Notes](https://help.obsidian.md/How+to/Use+callouts) style alerts/callouts:

```
> [!warning]
> This is a warning
```

instead of what you posted:
```
::: warning
This is a warning
:::
```

Basically want a colored background like you have [in your blog](https://blog.kdheepak.com/pandoc-lua-filter-for-alerts), but running it through Pandoc to generate PDFs. Do you have an idea of what to change? I read through some of the Lua documentation but I'm having a hard time finding what to change.

**Comment by kdheepak** [comment]
Hmm. You’ll have to use a `Quote` function filter instead of whatever I’ve used. And then check what the first line in the quote is and generate the appropriate HTML instead. 

It’s not too difficult and I’d normally be able to take a stab at it but I’m not at my computer and I won’t be able to for a little while (I have limited access to the internet for the next few weeks). 

You can probably post on the pandoc filter Google groups or on GitHub for help.
